ui/pages/confirmation/confirmation.js

```diff
@@ -176,6 +178,8 @@ export default function ConfirmationPage({
     isEqual,
   );
   const unapprovedTxsCount = useSelector(getUnapprovedTxCount);
+  const pendingApprovalFlows = useSelector(getPendingApprovalFlows);
```

Comment:
As this is an array rather than a simple value, do we need to use the comparison argument with `isEqual` to ensure it detects changes?